src/getClasses.js

```diff
@@ -6,29 +6,22 @@
  */
 export function getClasses( el )
 {
-  let classNames;
-
-  try
+  if( !el.hasAttribute( 'class' ) )
   {
-    classNames = el.classList.toString().split( ' ' );
+    return [];
   }
-  catch ( e )
-  {
-    if( !el.hasAttribute( 'class' ) )
-    {
-      return [];
-    }
 
+  try {
+    return Array.from(el.classList);
```

Comment:
@jeetiss if we can achieve it without polyfill would be better, to keep the size of the lib as less as possible :)